lib/Cron/Workers/KeyCheck/SendingServiceKeyCheck.php

```diff
@@ -40,10 +41,17 @@ public function getNextRunDate() {
   }
 
   public function checkKey() {
+    $wasPendingApproval = $this->servicesChecker->isMailPoetAPIKeyPendingApproval();
+
     $mssKey = $this->settings->get(Mailer::MAILER_CONFIG_SETTING_NAME)['mailpoet_api_key'];
     $result = $this->bridge->checkMSSKey($mssKey);
     $this->bridge->storeMSSKeyAndState($mssKey, $result);
     $this->bridge->updateSubscriberCount($result);
+
+    $isPendingApproval = $this->servicesChecker->isMailPoetAPIKeyPendingApproval();
+    if ($wasPendingApproval && !$isPendingApproval) {
+      MailerLog::resumeSending();
+    }
```

Comment:
I think it would be nice to add a test for this functionality. We already have an [integration test for this service](https://github.com/mailpoet/mailpoet/blob/master/tests/integration/Cron/Workers/KeyCheck/SendingServiceKeyCheckTest.php) so I guess we could add a test case for this.

Reply:
👍  I'll add a test.